DeckBuilder/DeckBuilder.cpp

```diff
@@ -9,12 +8,55 @@ DeckBuilder::DeckBuilder()
 
 void addCard(Card &cardToAdd)
 {
+        Logger logger;
+        
+        if(cardCount[cardToAdd]=<3 || cardCount[cardToAdd] == NULL)
+        {
+            cardCount[cardToAdd]++;
+            if (deck.numberOfCards >= 40)
```

Comment:
to zmienna 40, albo ta wyzej 3 mozna jakos nazwac

np. contexpr auto maxNumberOfCards = 40u;

 if (deck.numberOfCards >= maxNumberOfCards)